apps/docs/pages/guides/auth/social-login/auth-azure.mdx

```diff
@@ -77,7 +74,7 @@ async function signInWithAzure() {
   const { data, error } = await supabase.auth.signInWithOAuth({
     provider: 'azure',
     options: {
-      scopes: 'email',
+      scopes: 'email', // this is important
```

Comment:
i don't think this change is necessary, the admonition above already calls it out